Commit 1ae9a5d causes cuda_runtime.h not found

##### System information (version)
- OpenCV => Latest
- Operating System / Platform => Ubuntu 14.04
- Compiler => gcc 4.9.3
- CUDA => 7.5

##### Steps to reproduce

Write any code with the line

``````
#include "opencv2/opencv.hpp"
``````

git checkout 1ae9a5d

rebuild OpenCV with CUDA enabled

compile your code and you will see error something like 

``````
In file included from /usr/local/include/opencv2/cudalegacy.hpp:47:0,
                 from /usr/local/include/opencv2/opencv.hpp:78,
                 from /home/jason/repo/modules/test/include/jason/test/classA.h:8,
                 from /home/jason/repo/modules/test/src/classA.cpp:3:
/usr/local/include/opencv2/cudalegacy/NCV.hpp:52:26: fatal error: cuda_runtime.h: No such file or directory
 #include <cuda_runtime.h>
                          ^
compilation terminated.
``````

cuda_runtime.h exists in my machine at 

``````
/usr/local/cuda-7.5/include/cuda_runtime.h
/usr/local/cuda/include/cuda_runtime.h
``````
cuda is a symlink folder to cuda-7.5

Now go back to OpenCV and 

git checkout 29afb83

rebuild OpenCV again with CUDA enabled

compile your code and it will work.

Would be great to see it fixed as I have to temporarily disable my cronjobs that periodically pull the latest change of OpenCV

--- a/include/opencv2/opencv.hpp
+++ b/include/opencv2/opencv.hpp
@@ -56,42 +56,6 @@
 #ifdef HAVE_OPENCV_CALIB3D
 #include "opencv2/calib3d.hpp"
 #endif
-#ifdef HAVE_OPENCV_CUDAARITHM
-#include "opencv2/cudaarithm.hpp"
-#endif
-#ifdef HAVE_OPENCV_CUDABGSEGM
-#include "opencv2/cudabgsegm.hpp"
-#endif
-#ifdef HAVE_OPENCV_CUDACODEC
-#include "opencv2/cudacodec.hpp"
-#endif
-#ifdef HAVE_OPENCV_CUDAFEATURES2D
-#include "opencv2/cudafeatures2d.hpp"
-#endif
-#ifdef HAVE_OPENCV_CUDAFILTERS
-#include "opencv2/cudafilters.hpp"
-#endif
-#ifdef HAVE_OPENCV_CUDAIMGPROC
-#include "opencv2/cudaimgproc.hpp"
-#endif
-#ifdef HAVE_OPENCV_CUDALEGACY
-#include "opencv2/cudalegacy.hpp"
-#endif
-#ifdef HAVE_OPENCV_CUDAOBJDETECT
-#include "opencv2/cudaobjdetect.hpp"
-#endif
-#ifdef HAVE_OPENCV_CUDAOPTFLOW
-#include "opencv2/cudaoptflow.hpp"
-#endif
-#ifdef HAVE_OPENCV_CUDASTEREO
-#include "opencv2/cudastereo.hpp"
-#endif
-#ifdef HAVE_OPENCV_CUDAWARPING
-#include "opencv2/cudawarping.hpp"
-#endif
-#ifdef HAVE_OPENCV_CUDEV
-#include "opencv2/cudev.hpp"
-#endif
 #ifdef HAVE_OPENCV_FEATURES2D
 #include "opencv2/features2d.hpp"
 #endif

